vit_unetr_ssl/transforms.py

```diff
@@ -0,0 +1,224 @@
+            LoadImaged(keys=["image", "label"], image_only=False),
+            # Ensure that the channel dimension is the first dimension of the image tensor.
+            EnsureChannelFirstd(keys=["image", "label"]),
+            # Ensure that the image orientation is consistently RPI
+            Orientationd(keys=["image", "label"], axcodes="RPI"),
+            # Resample the images to a specified pixel spacing
+            # NOTE: spine interpolation with order=2 is spline, order=1 is linear
+            Spacingd(keys=["image", "label"], pixdim=(1.0, 1.0, 1.0), mode=(2, 1)),
+            # Normalize the intensity values of the image
+            NormalizeIntensityd(keys=["image"], nonzero=False, channel_wise=False),
+            # Remove background pixels to focus on regions of interest.
+            CropForegroundd(keys=["image", "label"], source_key="image"),
+            # Pad the image to a specified spatial size if its size is smaller than the specified dimensions
+            ResizeWithPadOrCropd(keys=["image", "label"], spatial_size=spatial_size),
+            # Randomly crop samples of a specified size
+            RandCropByPosNegLabeld(
+                keys=["image", "label"],
+                label_key="label",
+                spatial_size=spatial_size,
+                pos=1,
+                neg=1,
+                num_samples=4,  # if num_samples=4, then 4 samples/image are randomly generated
+                image_key="image",
+                image_threshold=0,
+            ),
+            # data-augmentation
+            # NOTE: the following transforms are based on contrast-agnostic data augmentation:
+            # https://github.com/ivadomed/model-seg-dcm/blob/nk/dcm-zurich-pretraining/monai/transforms.py
+            RandAffined(keys=["image", "label"], mode=(2, 1), prob=0.9,
+                        rotate_range=(-20. / 360 * 2. * np.pi, 20. / 360 * 2. * np.pi),
+                        # monai expects in radians
+                        scale_range=(-0.2, 0.2),
+                        translate_range=(-0.1, 0.1)),
+            Rand3DElasticd(keys=["image", "label"], prob=0.5,
+                           sigma_range=(3.5, 5.5),
+                           magnitude_range=(25., 35.)),
+            RandSimulateLowResolutiond(keys=["image"], zoom_range=(0.5, 1.0), prob=0.25),
+            RandAdjustContrastd(keys=["image"], gamma=(0.5, 3.), prob=0.5),  # this is monai's RandomGamma
+            RandBiasFieldd(keys=["image"], coeff_range=(0.0, 0.5), degree=3, prob=0.3),
+            RandGaussianNoised(keys=["image"], mean=0.0, std=0.1, prob=0.1),
+            RandGaussianSmoothd(keys=["image"], sigma_x=(0., 2.), sigma_y=(0., 2.), sigma_z=(0., 2.0),
+                                prob=0.3),
+            RandScaleIntensityd(keys=["image"], factors=(-0.25, 1), prob=0.15),
+            # this is nnUNet's BrightnessMultiplicativeTransform
+            RandFlipd(keys=["image", "label"], prob=0.3, ),
```

Comment:
oh man, taking these transforms exactly from `contrast-agnostic` is not a good idea! They heavily deform the image (useful for SC seg but not for lesion seg). We need much simpler transforms for lesion seg

Reply:
Agree. Indeed, I just copy-pasted the transforms. But note that these transforms are for the fine-tuned model, which can theoretically be any type of segmentation (SC, lesions, ...). But agree, I will accept your suggestions! Thanks!

Reply:
Committed in https://github.com/ivadomed/model-seg-dcm/pull/10/commits/a4182b5f91aa8693e617ae933fde3856fc8a2ad1